swarm/test/cli_test.go

```diff
@@ -0,0 +1,84 @@
+package swarmtest
+
+import (
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+// TestCLISwarmUp tests that running 'swarm up' makes the resulting file
+// available from all nodes via the HTTP API
+func TestCLISwarmUp(t *testing.T) {
```

Comment:
So these are already the kinds of tests i want the network testing framework to do. 
Here i assume your  objective is just to have basic tests for the CLI itself?

Just wonder how this will align with our scenario driven test framework and if it is worth the effort 
maintaining this. Let's have a chat